src/sqlancer/sqlite3/gen/ddl/SQLite3IndexGenerator.java

```diff
@@ -44,6 +44,7 @@ private SQLQueryAdapter create() throws SQLException {
         errors.add("non-deterministic use of date() in an index");
         errors.add("non-deterministic use of datetime() in an index");
         errors.add("The database file is locked");
+        errors.add("Abort due to constraint violation");
```

Comment:
Is this added error intended? I suspect that this might be for an older or more recent version of SQLite that we are currently testing?

Reply:
No, I think this was an artifact of a merge or something previous. I've removed it.